namerd/docs/storage.md

```diff
@@ -1,89 +1,99 @@
 # Storage
 
-*(for the [storage](config.md#storage) key)*
-
 A storage object configures the namerd dtabStore which stores and retrieves
 dtabs. This object supports the following params:
 
-* *kind* -- The name of the storage plugin.
-* *experimental* -- Set this to `true` to enable the storage if it is experimental.
-* *sotrage-specific parameters*.
+<aside class="notice">
+These parameters are available to the storage regardless of kind. Storage may also have kind-specific parameters.
+</aside>
+
+Key | Default Value | Description
+--- | ------------- | -----------
+kind | _required_ | Either `io.l5d.inMemory`, `io.l5d.k8s`, `io.l5d.zk`, `io.l5d.etcd` or `io.l5d.consul`.
+experimental | `false` | Set this to `true` to enable the storage if it is experimental.
+port | interface dependent | The port number on which to server the namer interface.
```

Comment:
I think this might be a copy/paste issue -- pretty sure there is not a port option for storage.  Maybe this should be `kind-specific`?